src/material/core/focus-indicators/_focus-indicators-theme.scss

```diff
@@ -5,43 +5,18 @@
 @mixin color($config-or-theme) {
   $config: theming.get-color-config($config-or-theme);
   $border-color: theming.get-color-from-palette(map.get($config, primary));
-  @include _mat-strong-focus-indicators-border-color($border-color);
-}
 
-/// Mixin that sets the color of the focus indicators.
-///
-/// @param {color|map} $theme-or-color
-///   If theme, focus indicators are set to the primary color of the theme. If
-///   color, focus indicators are set to that color.
-///
-/// @example
-///   .demo-dark-theme {
-///     @include mat-strong-focus-indicators-theme($dark-theme-map);
-///   }
-///
-/// @example
-///   .demo-red-theme {
-///     @include mat-strong-focus-indicators-theme(#f00);
-///   }
-// stylelint-disable-next-line material/theme-mixin-api
-@mixin theme($theme-or-color) {
-  @if meta.type-of($theme-or-color) != 'map' {
-    @include _mat-strong-focus-indicators-border-color($theme-or-color);
-  }
-  @else {
-    $theme: theming.private-legacy-get-theme($theme-or-color);
-    @include theming.private-check-duplicate-theme-styles($theme, 'mat-strong-focus-indicators') {
-      $color: theming.get-color-config($theme);
-      @if $color != null {
-        @include color($color);
-      }
-    }
+  .mat-focus-indicator::before {
+    border-color: $border-color;
   }
 }
 
-// Mixin that applies the border color for the focus indicators.
-@mixin _mat-strong-focus-indicators-border-color($color) {
-  .mat-focus-indicator::before {
-    border-color: $color;
+@mixin theme($theme-or-color-config) {
+  $theme: theming.private-legacy-get-theme($theme-or-color-config);
+  @include theming.private-check-duplicate-theme-styles($theme, 'mat-focus-indicators') {
```

Comment:
Note: The id to `private-check-duplicate-theme-styles` was changed from `mat-strong-focus-indicators` to `mat-focus-indicators` to address a linter error I was getting:

> Missing check for duplicative theme styles.  material/theme-mixin-api
> Please include the duplicate styles check mixin: theming.private-check-duplicate-theme-styles($theme, 'mat-focus-indicators')